Dockerfile

```
@@ -0,0 +1,16 @@
+## use a java 17 image
+FROM openjdk:17-oracle
+
+LABEL maintainer="Ivan Castillo <sacnavi@outlook.com>"
+
+## set app db credentials (default to cloud database on Atlas)
+ENV APP_DB=adoptpetdb
+ENV APP_USR=adoptpet
+ENV APP_PASS=rqIYZp0tmt75UaPH
+ENV APP_DB_URL=mongodb+srv://${APP_USR}:${APP_PASS}@cluster0.brcnj.mongodb.net/${APP_DB}
```

las primeras variables no se están usando, la última se sobreescribe, se puede poner algo genérico
```suggestion
ENV APP_DB_URL=mongodb://localhost:27017/db
```